zookeeper-server/src/main/java/org/apache/zookeeper/server/admin/Commands.java

```diff
@@ -743,7 +744,14 @@ public CommandResponse run(ZooKeeperServer zkServer, Map<String, String> kwargs)
                                 && voter.addr.equals(peer.getQuorumAddress())
                                 && voter.electionAddr.equals(peer.getElectionAddress())
                 );
+                response.put("myid", zkServer.getConf().getServerId());
+                response.put("is_leader", zkServer instanceof LeaderZooKeeperServer);
```

Comment:
We' better not use 'instanceof', can you please add a method to the base class and implement it with a default value of 'false'?

Reply:
@eolivelli 
- I just follow the implementation of command:`LeaderCommand` and I have no idea whether there is a better way

- > We' better not use 'instanceof', can you please add a method to the base class and implement it with a default value of 'false'?

   I may not get your idea,Could you plz give me more info?